modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -57,82 +57,149 @@ podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
 podman login registry.redhat.io
 ----
 +
-For more information about registry authentication, see https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
-
-. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation, by running the following commands:
+. Create an `ImageSetConfiguration` file to specify the resources that you want to mirror. For example:
++
+[source,terminal]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: ./mirror-output
+mirror:
+  helm:
+    repositories:
+      - name: openshift-charts
+        url: https://charts.openshift.io
+        charts:
+          - name: redhat-developer-hub
+            version: "{product-version}"
+----
+. Mirror the resources specified in the`ImageSetConfiguration` file directly to the target registry by running the following command:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal]
 ----
-podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version}
+oc-mirror --config=mirror-config.yaml file://mirror-archive
```

Comment:
```suggestion
oc-mirror --config=mirror-config.yaml docker://registry.example:5000
```

We are mirroring directly to the target registry in a partially disconnected environment. Otherwise, the images will never be pushed to the target mirror registry.
(This command is just an example - `registry.example:5000` should be replaced by the user)

Reply:
Also, as verification procedure, the `oc-mirror` command above should end up with something like this once done:
```
[...]
Writing image mapping to oc-mirror-workspace/results-1738070846/mapping.txt
Writing ICSP manifests to oc-mirror-workspace/results-1738070846
```

The user needs to keep track of the directory mentioned since this contains the Helm Chart to apply.